test/validation.js

```diff
@@ -57,6 +57,23 @@ describe('validation tests', function () {
         })
         .argv
     })
+
+    it('should ignore the --no- part if boolean negation is disabled', function (done) {
```

Comment:
rather than:

`it('should ignore the --no- part if boolean negation is disabled')`

I might put:

`it('does not treat --no- as a special case, if boolean negation is disabled')`,

Had to read the current test description a few times, I might just be nit-picking though :)